seedlet/functions.php

```diff
@@ -345,11 +345,14 @@ function seedlet_scripts() {
 	// Enqueue Google fonts
 	wp_enqueue_style( 'seedlet-fonts', seedlet_fonts_url(), array(), null );
 
-	// Theme variables
-	wp_enqueue_style( 'seedlet-variables-style', get_template_directory_uri() . '/assets/css/variables.css', array(), wp_get_theme()->get( 'Version' ) );
-
 	// Theme styles
-	wp_enqueue_style( 'seedlet-style', get_template_directory_uri() . '/style.css', array(), wp_get_theme()->get( 'Version' ) );
+	if (preg_match('~MSIE|Internet Explorer~i', $_SERVER['HTTP_USER_AGENT']) || (strpos($_SERVER['HTTP_USER_AGENT'], 'Trident/7.0; rv:11.0') !== false)) {
```

Comment:
Have you looked into one of the options suggested over here? https://gist.github.com/wpscholar/4947518#gistcomment-1669140

It looks like some of the Twenty- themes use a similar approach:

https://github.com/WordPress/twentyseventeen/blob/fa09211f54799dd03e8acda9d19108aafb1a8c77/functions.php#L286-L288

Reply:
Unfortunately IE10+ don't support conditional comments, otherwise we could use that approach.

Reply:
Ah ok, I see. What we have here is a bit lengthy, but I understand why it's necessary then.